common/lib/capa/capa/capa_problem.py

```diff
@@ -643,7 +643,7 @@ def do_targeted_feedback(self, tree):
         choice-level explanations shown to a student after submission.
         Does nothing if there is no targeted-feedback attribute.
         """
-        _ = self.capa_system.i18n.ugettext
+        _ = self.capa_system.i18n.gettext if six.PY3 else self.capa_system.i18n.ugettext
```

Comment:
We already created a utility function to do this, see [openedx/core/lib/edx_six.py](https://github.com/edx/edx-platform/blob/master/openedx/core/lib/edx_six.py) - `get_gettext()`.